python_modules/airline-demo/airline_demo/solids.py

```diff
@@ -491,14 +487,9 @@ def normalize_weather_na_values(_info, data_frame):
         )
     ],
     outputs=[OutputDefinition(SparkDataFrameType)],
-    config_field=Field(
-        types.ConfigDictionary(
-            name='BatchLoadDataToDatabaseFromSparkConfigType',
-            fields={
-                'table_name': Field(types.String, description=''),
-            }
-        )
-    )
+    config_field=Field(types.Dict(fields={
```

Comment:
what about adding some sugar so that we can write `Field({...})` for `Field(types.Dict(fields={...}))`?